api/app/tests/Behat/bootstrap/v2/Common/ReportTrait.php

```diff
@@ -61,7 +61,20 @@ private function submitSteps(string $ndrOrReport, int $reportId)
             }
         }
 
-        $this->clickLink('Continue');
+        if ('ndr' == $ndrOrReport) {
+            $this->clickLink('Continue');
+        } else {
+            $this->clickLink('Confirm contact details');
```

Comment:
If this is now a multiple step process then we should be separating them out into different functions and behat steps.

We should also be confirming what it is we expect to see on these pages and not just relying on there being a button